fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricMinecartComparatorLogicRegistry.java

```diff
@@ -0,0 +1,21 @@
+package net.fabricmc.fabric.api.object.builder.v1.entity;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.vehicle.AbstractMinecartEntity;
+
+public class FabricMinecartComparatorLogicRegistry {
+	private static final Map<EntityType<?>, MinecartComparatorLogic<?>> logics = new HashMap<>();
+
+	/* @Nullable */
+	@SuppressWarnings("unchecked") // just to appease typechecker, we checked in the entity type builder that the type of the logic exactly matches the entity type
```

Comment:
```suggestion
	@SuppressWarnings("unchecked")
```
This documentation imo isn't too useful as people know this is a trick for compiler

Reply:
Sure.